core/src/main/java/org/opensearch/sql/expression/function/PPLFuncImpTable.java

```diff
@@ -181,6 +181,7 @@ void populate() {
       registerOperator(ASIN, SqlStdOperatorTable.ASIN);
       registerOperator(ATAN, SqlStdOperatorTable.ATAN);
       registerOperator(ATAN2, SqlStdOperatorTable.ATAN2);
+      registerOperator(CEIL, SqlStdOperatorTable.CEIL);
```

Comment:
does it needed?

Reply:
Yes. the `ceil` function is missing and it is used in `CalciteFillNullCommandIT`. So I added it back.